vars/clientUtils.groovy

```diff
@@ -172,11 +172,15 @@ def setupIqePlugin(String plugin,String eggBranch=3.0){
         plugin_dir = 'iqe_insights_client'
         jenkins_credentials = 'settings_iqe_insights_client'
     }
-    else if(plugin == 'rhc'){
+    else if(plugin.contains('rhc')){
         git credentialsId: 'gitlab', url: 'https://gitlab.cee.redhat.com/insights-qe/iqe-rhc-plugin.git', branch: 'master'
         plugin_dir = 'iqe_rhc'
         jenkins_credentials = 'settings_iqe_rhc'
     }
+    else {
+        println("Unknown plugin string passed...")
```

Comment:
perhaps it could help you to display the value of `plugin`

Reply:
Thank you, I will add this into next PR.